components/_twig-components/functions/pl_attach-library.function.php

```diff
@@ -5,5 +5,20 @@
  */
 
 $function = new Twig_SimpleFunction('attach_library', function ($string) {
-  return $string;
-});
+  // Get Library Name from string.
+  $libraryName = substr($string, strpos($string, "/") + 1);
```

Comment:
This could return FALSE.

Reply:
when the library name is incorrect, it just doesn't load the file so I'm OK with this.